### PR TITLE
Adjust card feature documentation

### PR DESCRIPTION
**Pull request by piitaya**
## Proposed change

- [x] Add features option to thermostat card
- [x] Add features option to humidifier card
- [x] Update thermostat card screenshot to latest design
- [x] Update humidifier card screenshot to latest design
- [x] Rename tile features to card features and move the documentation to its own page
- [x] Add humidifier mode feature

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/frontend/pull/18732 https://github.com/home-assistant/frontend/pull/18711 https://github.com/home-assistant/frontend/pull/18709
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/dashboards/features.markdown [hunk 1]
```diff
@@ -0,0 +1,356 @@
+---
+title: "Features for dashboard cards"
+description: "Decorate your dashboard cards with quick controls."
+---
+
+Some dashboard cards have support for features.These widgets add quick controls to the card. Supported features depend on the card and entity capabilities. Multiple features can be added to a single card.
+
+<p class='img'><img src='/images/dashboards/features/screenshot-tile-feature-grid.png' alt="Screenshot of tile cards with a features.">
```
Comment (c0ffeeca7) [inline source/dashboards/features.markdown]:
```suggestion
<p class='img'><img src='/images/dashboards/features/screenshot-tile-feature-grid.png' alt="Screenshot of tile cards with features.">
```

File: source/dashboards/features.markdown [hunk 2]
```diff
@@ -0,0 +1,356 @@
...
+Screenshot of tile cards with a features.
```
Comment (c0ffeeca7) [inline source/dashboards/features.markdown]:
```suggestion
Screenshot of tile cards with features.
```

File: source/dashboards/features.markdown [hunk 3]
```diff
@@ -0,0 +1,356 @@
...
+</p>
+
+## Alarm modes
+
+Widget that display buttons to arm and disarm an [alarm](/integrations/alarm_control_panel).
```
Comment (c0ffeeca7) [inline source/dashboards/features.markdown]:
```suggestion
Widget that displays buttons to arm and disarm an [alarm](/integrations/alarm_control_panel).
```

File: source/dashboards/features.markdown [hunk 4]
```diff
@@ -0,0 +1,356 @@
...
+```
+
+{% configuration features %}
+type:
+  required: true
+  description: "`alarm-modes`"
+  type: string
+modes:
+  required: true
+  description: List of modes to show on the card. The list can contain `armed_home`, `armed_away`, `armed_night`, `armed_vacation`, `armed_custom_bypass` and `disarmed`.
```
Comment (c0ffeeca7) [inline source/dashboards/features.markdown]:
```suggestion
  description: List of modes to show on the card. The list can contain `armed_home`, `armed_away`, `armed_night`, `armed_vacation`, `armed_custom_bypass`, and `disarmed`.
```

File: source/dashboards/features.markdown [hunk 5]
```diff
@@ -0,0 +1,356 @@
...
+```
+
+{% configuration features %}
+type:
+  required: true
+  description: "`climate-hvac-modes`"
+  type: string
+hvac_modes:
+  required: true
+  description: List of modes to show on the card. The list can contain `auto`, `heat_cool`, `heat`, `cool`, `dry`, `fan_only` and `off`.
```
Comment (c0ffeeca7) [inline source/dashboards/features.markdown]:
```suggestion
  description: List of modes to show on the card. The list can contain `auto`, `heat_cool`, `heat`, `cool`, `dry`, `fan_only`, and `off`.
```

File: source/dashboards/features.markdown [hunk 6]
```diff
@@ -0,0 +1,356 @@
...
+```
+
+{% configuration features %}
+type:
+  required: true
+  description: "`vacuum-commands`"
+  type: string
+commands:
+  required: true
+  description: List of commands to show on the card. The list can contain `start_pause`, `stop`, `clean_spot`, `locate` and `return_home`.
```
Comment (c0ffeeca7) [inline source/dashboards/features.markdown]:
```suggestion
  description: List of commands to show on the card. The list can contain `start_pause`, `stop`, `clean_spot`, `locate`, and `return_home`.
```

File: source/dashboards/features.markdown [hunk 7]
```diff
@@ -0,0 +1,356 @@
...
+```
+
+{% configuration features %}
+type:
+  required: true
+  description: "`water-heater-operation-modes`"
+  type: string
+operation_modes:
+  required: true
+  description: List of modes to show on the card. The list can contain `electric`, `gas`, `heat_pump`, `eco`, `performance`, `high_demand` and `off`.
```
Comment (c0ffeeca7) [inline source/dashboards/features.markdown]:
```suggestion
  description: List of modes to show on the card. The list can contain `electric`, `gas`, `heat_pump`, `eco`, `performance`, `high_demand`, and `off`.
```

**Comment by piitaya** [comment]
Thank you @c0ffeeca7 🙏

File: source/dashboards/features.markdown [hunk 8]
```diff
@@ -0,0 +1,356 @@
+---
+title: "Features for dashboard cards"
+description: "Decorate your dashboard cards with quick controls."
+---
+
+Some dashboard cards have support for features.These widgets add quick controls to the card. Supported features depend on the card and entity capabilities. Multiple features can be added to a single card.
```
Comment (c0ffeeca7) [inline source/dashboards/features.markdown]:
```suggestion
Some dashboard cards have support for features. These widgets add quick controls to the card. Supported features depend on the card and entity capabilities. Multiple features can be added to a single card.
```

**Review by c0ffeeca7 — COMMENTED** [review]
found one more :muscle:

**Review by c0ffeeca7 — APPROVED** [review]
Awesome feature :star_struck: Looking slick.
Thank you, @piitaya :+1:

**Pull request merged**